lib/simp/cli/config/items/action/warn_verify_user_access_after_bootstrap_action.rb

```diff
@@ -0,0 +1,59 @@
+require_relative '../action_item'
+require_relative '../data/cli_local_priv_user'
+
+module Simp; end
+class Simp::Cli; end
+module Simp::Cli::Config
+  class Item::WarnVerifyUserAccessAfterBootstrapAction < ActionItem
+
+    def initialize(puppet_env_info = DEFAULT_PUPPET_ENV_INFO)
+      super(puppet_env_info)
+      @key         = 'login::verify::access'
+      @description = 'Verify access after `simp bootstrap`'
+      @category    = :sanity_check
+    end
+
+    def apply
+      username = get_item('cli::local_priv_user').value
+      warning_message = <<~DOC
+
+        #########################################################################
+        #                            CRITICAL WARNING                           #
+        #########################################################################
+
+        **After** `simp bootstrap` but **before** you reboot the server or
+        close the terminal, it is imperative that you verify user '#{username}'
+        access as follows:
+
+        Using a NEW SSH SESSION OR TERMINAL (do NOT close your working session)
+
+        * Log in as '#{username}'
```

Comment:
Maybe make this an explicity `ssh <username>@<hostname>` so that it's more like a command?